plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -0,0 +1,222 @@
+            content={
+              <span>
+                {'You can run Docker containers with both container runtimes. The Universal Container Runtime is better supported in DC/OS. '}
+                <a href={MetadataStore.buildDocsURI('/usage/containerizers/')} target="_blank">More information</a>.
+              </span>
+            }
+            interactive={true}
+            maxWidth={300}
+            scrollContainer=".gm-scroll-view"
+            wrapText={true}>
+              <Icon color="grey" id="ring-question" size="mini" family="mini" />
+          </Tooltip>
+        </h3>
+        <p>The container runtime is responsible for running your service. We support the Mesos and Docker containerizers.</p>
+        <FormGroup showError={Boolean(containerErrors.type)}>
+          {selections}
+          <FieldError>{containerErrors.type}</FieldError>
+        </FormGroup>
+      </div>
+    );
+  }
+
+  getIDHelpBlock() {
+    return (
+      <span>
+        {'Include the path to your service, if applicable. E.g. /dev/tools/my-service. '}
+        <a href="https://mesosphere.github.io/marathon/docs/application-groups.html" target="_blank">
+          More information
+        </a>.
+      </span>
+    );
+  }
+
+  render() {
+    let {data, errors} = this.props;
+    let advancedSettingsIcon = 'triangle-right';
+    if (this.state.showAdvancedSettings) {
+      advancedSettingsIcon = 'triangle-down';
+    }
+
+    return (
+      <div className="form flush-bottom">
+        <div className="form-row-element">
+          <h2 className="form-header flush-top short-bottom">
+            Services
+          </h2>
+          <p>
+            Configure your service below. Start by giving your service a name.
+          </p>
+        </div>
+
+        <div className="flex row">
+          <FormGroup
+            className="column-8"
+            required={true}
+            showError={Boolean(errors.id)}>
+            <FieldLabel>
+              Service Name
+            </FieldLabel>
+            <FieldInput
+              name="id"
+              type="text"
+              value={data.id} />
+            <FieldHelp>{this.getIDHelpBlock()}</FieldHelp>
+            <FieldError>{errors.id}</FieldError>
+          </FormGroup>
+
+          <FormGroup
+            className="column-4"
+            showError={Boolean(errors.instances)}>
+            <FieldLabel>
+              Instances
+            </FieldLabel>
+            <FieldInput
+              name="instances"
+              min={0}
+              type="number"
+              value={data.instances} />
+            <FieldError>{errors.instances}</FieldError>
+          </FormGroup>
+        </div>
+
+        <p>
+          <a onClick={this.toggleAdvancedSettings}>
+            <Icon
+              id={advancedSettingsIcon}
+              color="purple"
+              family="mini"
+              size="mini" />
+            Advanced Service Settings
+          </a>
+        </p>
```

Comment:
Let's not use `p` tags for the spacing. We'll probably want to create a new component for this:
![](https://cl.ly/2F0s200N1G0I/Screen%20Shot%202016-11-15%20at%2010.10.30%20AM.png)